### ISSUE TITLE
Generate additional training data

### ISSUE THREAD

**Issue by tschimbr** [eonum/medcodelearn#41]
Always compare accuraccy on DRG classification with test set of original data.

1. Random sampling from distribution of number of DX and SRG
2. Random sampling from a priori distribution of DX and SRG
3. Start with PDX a priori sampling. Continue with sampling from proposal model distribution.
4. GANs
